test/integration_test/kubevirt_test.go

```diff
@@ -246,3 +258,65 @@ func createDatadiskTemplates(t *testing.T) error {
 	}
 	return nil
 }
+
+func kubevirtVMScaledDeployAndValidate(t *testing.T, instanceID, appKey string, scale int, allCtx *([]*scheduler.Context)) {
+	var wg sync.WaitGroup
+	logrus.Infof("Deploying %d VMs and validating them.", scale)
+	for i := 1; i <= scale; i++ {
+		logrus.Infof("Deploying VM: %d", i)
+		wg.Add(1)
+		ctxs := kubevirtVMsDeployAndValidate(
+			t,
+			instanceID+"-"+fmt.Sprintf("%d", i),
+			[]string{appKey},
+			&wg,
+		)
+		logrus.Infof("Validated VM: %d", i)
+		*allCtx = append(*allCtx, ctxs...)
+	}
+	wg.Wait()
+}
+
+func createClusterroleDataVolume(clusterroleName string) error {
+	clusterRoleDatavolume := &rbacv1.ClusterRole{
+		ObjectMeta: meta.ObjectMeta{
+			Name: clusterroleName,
+		},
+		Rules: []rbacv1.PolicyRule{{
+			Verbs:     []string{"*"},
+			APIGroups: []string{"*"},
+			Resources: []string{"*"},
+		},
```

Comment:
Hmmm... this is all powerful cluster role. It is not just for the data volume. So, the name is misleading. But should we be doing this? Could this potentially hide bugs?